engine/src/main/java/org/hibernate/validator/internal/constraintvalidators/bv/NegativeValidatorForNumber.java

```diff
@@ -0,0 +1,89 @@
+/*
+ * Hibernate Validator, declare and validate application constraints
+ *
+ * License: Apache License, Version 2.0
+ * See the license.txt file in the root directory or <http://www.apache.org/licenses/LICENSE-2.0>.
+ */
+package org.hibernate.validator.internal.constraintvalidators.bv;
+
+import java.math.BigDecimal;
+import java.math.BigInteger;
+
+import javax.validation.ConstraintValidator;
+import javax.validation.ConstraintValidatorContext;
+import javax.validation.constraints.Negative;
+
+/**
+ * Check that the number being validated is negative.
+ *
+ * @author Hardy Ferentschik
+ * @author Xavier Sosnovsky
```

Comment:
It's based on DecimalMin/Max validators so I let the name of the original authors.